buildconfig/CMake/DarwinSetup.cmake

```diff
@@ -79,11 +79,11 @@ if ( NOT TARGET mantidpython )
   else ()
     set ( PARAVIEW_PYTHON_PATHS "" )
   endif ()
-  configure_file ( ${CMAKE_MODULE_PATH}/Packaging/osx/mantidpython_osx ${CMAKE_CURRENT_BINARY_DIR}/mantidpython_osx @ONLY )
+  configure_file ( ${CMAKE_MODULE_PATH}/Packaging/osx/mantidpython.in ${CMAKE_BINARY_DIR}/bin/mantidpython @ONLY )
```

Comment:
Can we use `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/mantidpython` instead of `${CMAKE_BINARY_DIR}/bin/mantidpython` here? This avoids us having to define where the `bin` directory is in multiple locations in the cmake.